frontend/src/Components/Headers/index.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useContext } from "react";
+import Note from "plaid-threads/Note";
+import InlineLink from "plaid-threads/InlineLink";
+
+import Link from "../Link";
+import Context from "../../Context";
+
+import styles from "./index.module.scss";
+
+interface Props {
+  currentPath: string;
+}
+
+const Header = (props: Props) => {
+  const {
+    itemId,
+    accessToken,
+    linkToken,
+    linkSuccess,
+    isItemAccess,
+  } = useContext(Context);
+
+  return (
+    <div className={styles.grid}>
+      <h3 className={styles.title}>Plaid Quickstart</h3>
+
+      {!linkSuccess && (
+        <>
+          <h4 className={styles.subtitle}>
+            A sample end-to-end integration with Plaid
+          </h4>
+          <p className={styles.introPar}>
+            The Plaid flow begins when your user wants to connect their bank
+            account to your app. Simulate this by clicking the button below to
+            launch Link - the client-side component that your users will
+            interact with in order to link their accounts to Plaid and allow you
+            to access their accounts via the Plaid API.
+          </p>
+          {/* message if backend is not running and there is no link token */}
+          {linkToken == null && (
+            <Note error solid className={styles.runBackendError}>
+              Unable to fetch link_token: please make sure your backend server
+              is running.
+            </Note>
+          )}
+          {linkToken != null && (
+            <div className={styles.linkButton}>
+              <Link currentPath={props.currentPath} />
+            </div>
+          )}
+        </>
+      )}
+      {linkSuccess && (
+        <>
+          {isItemAccess && (
+            <h4 className={styles.subtitle}>
+              Congrats! By linking an account, you have created an{" "}
+              <InlineLink
+                href="http://plaid.com/docs/quickstart/glossary/#item"
+                target="_blank"
+              >
+                Item
+              </InlineLink>
+              .
+            </h4>
+          )}
+          {!isItemAccess && (
+            <h4 className={styles.subtitle}>
+              Unable to create an <span className={styles.itemText}>Item</span>.
+              Please check your backend server
+            </h4>
```

Comment:
Just a UX thought: we have two different ways of showing an error message to the user. This one does not look like an error and is easy to miss since it looks the same as the default header text. Could we reuse the nice red error message that we see when generating a link token fails?

Reply:
done